redshift_sqlalchemy/dialect.py

```diff
@@ -264,6 +527,167 @@ def create_connect_args(self, *args, **kwargs):
         default_args.update(cparams)
         return cargs, default_args
 
+    @util.memoized_property
+    def _isolation_lookup(self):
+        extensions = __import__('psycopg2.extensions').extensions
```

Comment:
Remove, see if you need to bump the lower limit of sqlalchemy etc
On 26 Aug 2015 19:46, "Jeff Klukas" <notifications@github.com> wrote:

> In redshift_sqlalchemy/dialect.py
> <https://github.com/graingert/redshift_sqlalchemy/pull/17#discussion_r38019806>
> :
>
> > @@ -264,6 +527,167 @@ def create_connect_args(self, *args, **kwargs):
> >          default_args.update(cparams)
> >          return cargs, default_args
> >
> > +    @util.memoized_property
> > +    def _isolation_lookup(self):
> > +        extensions = __import__('psycopg2.extensions').extensions
>
> Tests all pass with this method removed. Looking at history, it dates back
> to 2013 in this Redshift dialect, and there have been significant changes
> to the same method in the psycopg2 dialect since then.
>
> I'm in favor of removal.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/graingert/redshift_sqlalchemy/pull/17/files#r38019806>
> .
>